assets/openshift-apiserver/openshift-apiserver-deployment.yaml

```diff
@@ -90,6 +90,26 @@ spec:
         - "--requestheader-group-headers=X-Remote-Group"
         - "--requestheader-extra-headers-prefix=X-Remote-Extra-"
         - "--client-ca-file=/etc/kubernetes/config/serving-ca.crt"
+        livenessProbe:
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            path: healthz
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          timeoutSeconds: 10
+          failureThreshold: 3
+          successThreshold: 1
+        readinessProbe:
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            path: healthz
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          timeoutSeconds: 1
+          failureThreshold: 10
```

Comment:
The readiness probe should fire before the liveness probe.  Waiting for 10 failures means that readiness would likely never first since liveness would probably fail first and restart the container.  I suggest switching the value between the readiness and liveness probes.

Reply:
updated failureThreshold to 3;   initalDelaySeconds is less than that of liveness, so should fire first